### DISCUSSION TITLE
These is little information in the telegram bot

### DISCUSSION THREAD

**Discussion by Homavi88** [masterking32/MasterHamsterKombatBot#395] [Q&A]
Is it possible to add more information from the terminal to the telegram bot 

sorry for the off topic question

**Comment by copycatcodex** [comment]
What do you mean by bot log/report and want to send it to Telegram?

**Comment by Microzibert** [comment]
> Is it possible to add more information from the terminal to the telegram bot
> 
> sorry for the off topic question

why? I use telegram only to be sure that the bot works and nothing is broken. If I need to get more detailed information, I open an ssh connection and look at the logs from anywhere in the world
